Gemfile.lock

```diff
@@ -120,6 +120,9 @@ GEM
     diff-lcs (1.3)
     dnsruby (1.60.2)
     docile (1.3.1)
+    em-websocket (0.3.8)
```

Comment:
WebSocket will be used in future for establishing bidirectional communication between meterpreter terminal (using xterm.js as a front end)  and Sinatra web-socket. I haven't used this gem yet but it will be useful in the future.